modules/linear_boltzmann_solvers/discrete_ordinates_solver/lbs_discrete_ordinates_solver.cc

```diff
@@ -654,55 +653,66 @@ DiscreteOrdinatesSolver::ComputeBalance()
     const auto& IntV_shapeI = fe_intgrl_values.intV_shapeI;
     const auto& IntS_shapeI = fe_intgrl_values.intS_shapeI;
 
-    // Inflow
-    // This is essentially an integration over
-    // all faces, all angles, and all groups.
-    // Only the cosines that are negative are
-    // added to the integral.
+    // Inflow: This is essentially an integration over all faces, all angles, and all groups. For
+    // non-reflective boundaries, only the cosines that are negative are added to the inflow
+    // integral. For reflective boundaries, inflow = outflow.
```

Comment:
This is ONLY true in the math world. In practice, there are iterations, etc., so that equality implies that there are the same but it is NOT true in a code. Please amend the comment. Maybe state, For reflective boundaries, it is expected that,upon convergence, inflow = outflow (within numerical tolerances set by the user).